docker/tests/test_zap_hooks.py

```diff
@@ -0,0 +1,83 @@
+import unittest
+from datetime import datetime
+from contextlib import contextmanager
+import zap_common
+
+
+class _MockHooks(object):
+
+    def __init__(self):
+        self.called = 0
+  
+
+    def zap_started(self, zap, target):
+        self.called += 1
+        return zap,
+
+
+class TestZapHooks(unittest.TestCase):
+
+    @contextmanager
+    def assertNotRaises(self, exc_type=Exception):
```

Comment:
Raised instead of Raises?

Reply:
I was trying to be consistent (and maybe illogical?) with the language of  `assertRaises` from stdlib but am not opposed to changing it.